go.mod

```diff
@@ -0,0 +1,35 @@
+module sigs.k8s.io/scheduler-plugins
+
+go 1.13
+
+require (
+	github.com/google/go-cmp v0.3.0
+	k8s.io/api v0.18.0
+	k8s.io/apimachinery v0.18.0
+	k8s.io/client-go v0.18.0
+	k8s.io/kubernetes v1.18.0
```

Comment:
We might need to use a dash in some cases. For example, assume there are bugs in this repository and we need to release a new version that is tied to k/k 1.18.1.
Then we should tag it as 1.18.1-2

Reply:
@Huang-Wei the plan sounds good to me.

Reply:
Good point.

BTW: I  checked [semver](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions). It seems `-` is used to present "ahead of" a release, e.g., k8s uses `1.18.0-alpha.0` to tag an early release ahead of 1.18.0. So we should following that as well, rather than using `1.18.1-2`, we should use `1.18.1+patch1` - `+` here represents the "post" semantics.